main/portaudio/template.py

```diff
@@ -1,7 +1,8 @@
 pkgname = "portaudio"
 pkgver = "19.7.0"
-pkgrel = 0
+pkgrel = 1
 build_style = "gnu_configure"
+configure_args = ["--enable-cxx"]
```

Comment:
```suggestion
configure_args = ["--enable-cxx"]
configure_gen = ["autoreconf", "-if", ".", "bindings/cpp"]
```

delete the pre_configure below